api/src/services/opportunities_v1/get_opportunity.py

```diff
@@ -8,7 +8,13 @@
 from src.api.route_utils import raise_flask_error
 from src.db.models.agency_models import Agency
 from src.db.models.opportunity_models import Opportunity, OpportunityAttachment, OpportunitySummary
-from src.util.file_util import pre_sign_file_location
+from src.util.env_config import PydanticBaseEnvConfig
+from src.util.file_util import convert_public_s3_to_cdn_url, pre_sign_file_location
+
+
+class AttachmentConfig(PydanticBaseEnvConfig):
+    # If the CDN URL is set, we'll use it instead of pre-signing the file locations
+    cdn_url: str | None = None
```

Comment:
This doesn't need to be in capslock?

Reply:
[Pydantic](https://docs.pydantic.dev/latest/concepts/pydantic_settings/#case-sensitivity) will look for `CDN_URL` as an env var. We sometimes also give values an alias if we want a cleaner python name, but doesn't matter too much here.